message/tests.py

```diff
@@ -0,0 +1,78 @@
+from django.utils import timezone
+from .models import Message
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+from .models import AuthorOptions
+import pytest
+
+AUTHOR = 'testuser@gmail.com'
+DOG_OWNER_ID = '123456789'
+DAYCARE_ID = '987654321'
+DATE = timezone.now()
+
+
+@pytest.fixture
+def dogowner_message1():
+    return Message.create(author=AuthorOptions.DogOwner,
+                          dogowner_id=DogOwner.objects.get(pk=1),
+                          daycare_id=DayCare.objects.get(pk=1),
+                          text='Hello this is the test message1 from owner to day care')
+
+
+@pytest.fixture
+def daycare_replay_to_dogonwer_message1():
+    return Message.create(author=AuthorOptions.DayCare,
+                          dogowner_id=DogOwner.objects.get(pk=1),
+                          daycare_id=DayCare.objects.get(pk=1),
+                          text='This is reply to first message from daycare to owner')
+
+
+@pytest.fixture
+def daycare_message2():
+    return Message.create(author=AuthorOptions.DayCare,
+                          dogowner_id=DogOwner.objects.get(pk=2),
+                          daycare_id=DayCare.objects.get(pk=2),
+                          text='Hello this is the test message2 from daycare to owner')
+
+
+@pytest.fixture
+def dogowner_replay_to_daycare_message2():
+    return Message.create(author=AuthorOptions.DogOwner,
```

Comment:
if this feature is supposed to be a chat, we don't need this reply system. messages with dates will do the trick (display the messages by their date, one after the other)